addons/l10n_sg/data/l10n_sg_chart_data.xml

```diff
@@ -740,7 +740,7 @@
         </record>
 
         <record model="account.account.template" id="account_account_796">
-            <field name="name">Sales Tax Account 7% SR</field>
+            <field name="name">Sales Tax Account 8% SR</field>
```

Comment:
They use the same account but I found no documentation specifying the name format. So to me renaming every account to remove the percent thing is ok. It's more readable (since they share the same account) and it will be future proof.